src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -114,19 +106,22 @@ const ItemFilters = ({
           renderMultiSelect={renderMultiSelect}
         />
         <Box minWidth="440">
-          <Label id="year-filter-label" htmlFor="year-filter">
+          <Label id="year-filter-label" htmlFor="searchbar-form-year-filter">
```

Comment:
If you're going to update `htmlFor` here, then the id for `SearchBar` should also be updated.

Reply:
I changed this because that what gets autogenerated by the DS, the might be a more robust way to fix that.